aten/src/ATen/native/LinearAlgebra.cpp

```diff
@@ -556,6 +556,13 @@ Tensor& dot_out(Tensor& result, const Tensor& self, const Tensor& tensor) {
   return result.fill_(self.dot(tensor));
 }
 
+Tensor& vdot_out(Tensor& result, const Tensor& self, const Tensor& tensor) {
+  result.resize_({});
```

Comment:
Use:

https://github.com/pytorch/pytorch/blob/dae94ed02299cd1e9840ce6e49ea142b4d118b2d/aten/src/ATen/native/Resize.cpp#L9

to resize outputs

Reply:
Adding it to `vdot_out`.

Should I also use it for `dot_out`?

https://github.com/pytorch/pytorch/blob/fa6b34b54c731938327c8e30e08b287a10b86b0a/aten/src/ATen/native/LinearAlgebra.cpp#L552-L557

Seems like a small change and the tests are already there for out variant.

Reply:
Please do! Thank you.